src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -514,17 +474,36 @@ def _get_input_dataset_from_synthetic(
         prompt_tokens_mean: int,
         prompt_tokens_stddev: int,
         num_of_output_prompts: int,
+        image_width_mean: int,
```

Comment:
Yea this isn't scalable.
I won't block this PR but we NEED a better long term solution.

Reply:
regardless of the above point, this is awesome work.
thanks for tackling this.